client/template.go

```diff
@@ -125,6 +129,21 @@ func (payload TemplateCreatePayload) Validate() error {
 		return errors.New("must supply terragrunt version")
 	}
 
+	if payload.IsTerragruntRunAll {
+		if payload.Type != "terragrunt" {
+			return errors.New(`can't set is_terragrunt_run_all to "true" for non-terragrunt template`)
+		}
+
+		c, _ := semver.NewConstraint(">= 0.28.1")
```

Comment:
I added semver library to compare versions and make sure the terragrunt version is above or equal 0.28.1